tactical-microgrid-standard/common/Handshaking.cpp

```diff
@@ -10,6 +10,10 @@
 #include <dds/DCPS/transport/framework/TransportRegistry.h>
 #include <dds/DCPS/transport/framework/TransportConfig.h>
 #include <dds/DCPS/transport/framework/TransportInst.h>
+#if defined (ACE_AS_STATIC_LIBS)
+# include <dds/DCPS/RTPS/RtpsDiscovery.h>
+# include <dds/DCPS/transport/rtps_udp/RtpsUdp.h>
+#endif
```

Comment:
We can use `StaticIncludes.h`. It will automatically initialize security if we linked it. Also I'm not sure how this works outside an executable C++, but usually these go in the executable source.